source/common/common/mutex_tracer_impl.cc

```diff
@@ -0,0 +1,42 @@
+#include "common/common/mutex_tracer_impl.h"
+
+#include <iostream>
+#include <memory>
+
+#include "common/common/assert.h"
+
+#include "absl/synchronization/mutex.h"
+
+namespace Envoy {
+
+MutexTracerImpl* MutexTracerImpl::singleton_ = nullptr;
+
+MutexTracerImpl* MutexTracerImpl::getOrCreateTracer() {
```

Comment:
I'll add a comment for `singleton_`. Maybe I'm not thinking of the other way, but I like storing a pointer so that getOrCreate can check for `==nullptr`.

Reply:
Thanks, resolved in 7602f67.